docs/workforce-management-framework/topics/con-introducing-raincatcher.adoc

```diff
@@ -1,23 +1,28 @@
 [id='con-introducing-raincatcher-{chapter}']
 = Introducing {WFM-RC-NameLong}
-
 //downstream == {WFM-RC-NameLong} is a Workforce Management Platform and is also referred to as {WFM-RC-NameShort}.
+
 {WFM-RC-NameLong} is a Workforce Management Platform.
-The latest version of {WFM-RC-NameShort} is {WFM-RC-Version}.
 
```

Comment:
{WFM-RC-NameShort} connects a business’ back-office with its fleet of mobilized employees.
{WFM-RC-NameShort} leverages commoditized mobile hardware, making use of each employee’s mobile phone.